### PR TITLE
Use assertj fluent style in flowable-engine.  Part 3

### PR DESCRIPTION
**Pull request by dbmalkovsky**
This finishes the `org.flowable.engine.test.api.event` package.

Continuing quest to use only a single style in each file and in the module.

### CODE REVIEW

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/VariableEventsTest.java [hunk 1]
```diff
@@ -100,18 +97,21 @@ public void testProcessInstanceVariableEvents() throws Exception {
...
+                        FlowableEngineEventType.VARIABLE_UPDATED,
+                        FlowableEngineEventType.VARIABLE_DELETED,
+                        FlowableEngineEventType.VARIABLE_DELETED
+                );
         listener.clearEventsReceived();
 
         // Delete nonexistent variable should not dispatch event
         runtimeService.removeVariable(processInstance.getId(), "unexistingVariable");
-        assertTrue(listener.getEventsReceived().isEmpty());
+        assertThat(listener.getEventsReceived().isEmpty()).isTrue();
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/VariableEventsTest.java]:
```suggestion
        assertThat(listener.getEventsReceived()).isEmpty();
```

Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/VariableEventsTest.java]: Fixed.

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java [hunk 2]
```diff
@@ -113,227 +110,228 @@ public void testProcessInstanceEvents() throws Exception {
...
-        assertEquals(FlowableEngineEventType.PROCESS_STARTED, event.getType());
-        assertEquals(subProcessInstanceId, event.getProcessInstanceId());
-        assertEquals(subProcessDefinitionId, event.getProcessDefinitionId());
-        assertTrue(event instanceof FlowableProcessStartedEvent);
-        assertEquals(processDefinitionId, ((FlowableProcessStartedEvent) event).getNestedProcessDefinitionId());
-        assertEquals(processInstance.getId(), ((FlowableProcessStartedEvent) event).getNestedProcessInstanceId());
+        assertThat(event.getType()).isEqualTo(FlowableEngineEventType.PROCESS_STARTED);
+        assertThat(event.getProcessInstanceId()).isEqualTo(subProcessInstanceId);
+        assertThat(event.getProcessDefinitionId()).isEqualTo(subProcessDefinitionId);
+        assertThat(event instanceof FlowableProcessStartedEvent).isTrue();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java]: @filiphr Fixed this and others; thanks.

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java [hunk 3]
```diff
@@ -394,122 +392,138 @@ public void testProcessCompleted_ParallelGatewayNoEnd() throws Exception {
...
+        assertThat(processCancelledEvents).as("ActivitiEventType.PROCESS_CANCELLED was expected 1 time.").hasSize(1);
         FlowableCancelledEvent processCancelledEvent = (FlowableCancelledEvent) processCancelledEvents.get(0);
-        assertTrue("The cause has to be the same as deleteProcessInstance method call", FlowableCancelledEvent.class.isAssignableFrom(processCancelledEvent.getClass()));
-        assertEquals("The process instance has to be the same as in deleteProcessInstance method call", processInstance.getId(), processCancelledEvent.getProcessInstanceId());
-        assertEquals("The execution instance has to be the same as in deleteProcessInstance method call", processInstance.getId(), processCancelledEvent.getExecutionId());
-        assertEquals("The cause has to be the same as in deleteProcessInstance method call", "delete_test", processCancelledEvent.getCause());
+        assertThat(FlowableCancelledEvent.class.isAssignableFrom(processCancelledEvent.getClass()))
+                .as("The cause has to be the same as deleteProcessInstance method call").isTrue();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java]: Fixed; thanks.

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java [hunk 4]
```diff
@@ -394,122 +392,138 @@ public void testProcessCompleted_ParallelGatewayNoEnd() throws Exception {
...
 
         List<FlowableEvent> taskCancelledEvents = listener.filterEvents(FlowableEngineEventType.ACTIVITY_CANCELLED);
-        assertEquals("ActivitiEventType.ACTIVITY_CANCELLED was expected 1 time.", 1, taskCancelledEvents.size());
+        assertThat(taskCancelledEvents).as("ActivitiEventType.ACTIVITY_CANCELLED was expected 1 time.").hasSize(1);
         FlowableActivityCancelledEvent activityCancelledEvent = (FlowableActivityCancelledEvent) taskCancelledEvents.get(0);
-        assertTrue("The cause has to be the same as deleteProcessInstance method call", FlowableActivityCancelledEvent.class.isAssignableFrom(activityCancelledEvent.getClass()));
-        assertEquals("The process instance has to be the same as in deleteProcessInstance method call", processInstance.getId(), activityCancelledEvent.getProcessInstanceId());
-        assertEquals("The cause has to be the same as in deleteProcessInstance method call", "delete_test", activityCancelledEvent.getCause());
+        assertThat(FlowableActivityCancelledEvent.class.isAssignableFrom(activityCancelledEvent.getClass()))
+                .as("The cause has to be the same as deleteProcessInstance method call").isTrue();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java]: Ditto

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java [hunk 5]
```diff
@@ -394,122 +392,138 @@ public void testProcessCompleted_ParallelGatewayNoEnd() throws Exception {
...
+        assertThat(processCancelledEvents).as("ActivitiEventType.PROCESS_CANCELLED was expected 2 times.").hasSize(2);
         FlowableCancelledEvent processCancelledEvent = (FlowableCancelledEvent) processCancelledEvents.get(0);
-        assertTrue("The cause has to be the same as deleteProcessInstance method call", FlowableCancelledEvent.class.isAssignableFrom(processCancelledEvent.getClass()));
-        assertEquals("The process instance has to be the same as in deleteProcessInstance method call", subProcess.getId(), processCancelledEvent.getProcessInstanceId());
-        assertEquals("The execution instance has to be the same as in deleteProcessInstance method call", subProcess.getId(), processCancelledEvent.getExecutionId());
-        assertEquals("The cause has to be the same as in deleteProcessInstance method call", "delete_test", processCancelledEvent.getCause());
+        assertThat(FlowableCancelledEvent.class.isAssignableFrom(processCancelledEvent.getClass()))
+                .as("The cause has to be the same as deleteProcessInstance method call").isTrue();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java]: Ditto

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java [hunk 6]
```diff
@@ -394,122 +392,138 @@ public void testProcessCompleted_ParallelGatewayNoEnd() throws Exception {
...
+                .isEqualTo(subProcess.getId());
+        assertThat(processCancelledEvent.getCause()).as("The cause has to be the same as in deleteProcessInstance method call").isEqualTo("delete_test");
 
         processCancelledEvent = (FlowableCancelledEvent) processCancelledEvents.get(1);
-        assertTrue("The cause has to be the same as deleteProcessInstance method call", FlowableCancelledEvent.class.isAssignableFrom(processCancelledEvent.getClass()));
-        assertEquals("The process instance has to be the same as in deleteProcessInstance method call", processInstance.getId(), processCancelledEvent.getProcessInstanceId());
-        assertEquals("The execution instance has to be the same as in deleteProcessInstance method call", processInstance.getId(), processCancelledEvent.getExecutionId());
-        assertEquals("The cause has to be the same as in deleteProcessInstance method call", "delete_test", processCancelledEvent.getCause());
+        assertThat(FlowableCancelledEvent.class.isAssignableFrom(processCancelledEvent.getClass()))
+                .as("The cause has to be the same as deleteProcessInstance method call").isTrue();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java]: Ditto

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java [hunk 7]
```diff
@@ -394,122 +392,138 @@ public void testProcessCompleted_ParallelGatewayNoEnd() throws Exception {
...
+        assertThat(activityCancelledEvent.getCause()).as("The cause has to be the same as in deleteProcessInstance method call").isEqualTo("delete_test");
 
         activityCancelledEvent = (FlowableActivityCancelledEvent) taskCancelledEvents.get(1);
-        assertTrue("The cause has to be the same as deleteProcessInstance method call", FlowableActivityCancelledEvent.class.isAssignableFrom(activityCancelledEvent.getClass()));
-        assertEquals("The process instance has to point to the main process", processInstance.getId(), activityCancelledEvent.getProcessInstanceId());
-        assertEquals("expect callActivity type", "callActivity", activityCancelledEvent.getActivityType());
-        assertEquals("The cause has to be the same as in deleteProcessInstance method call", "delete_test", activityCancelledEvent.getCause());
+        assertThat(FlowableActivityCancelledEvent.class.isAssignableFrom(activityCancelledEvent.getClass()))
+                .as("The cause has to be the same as deleteProcessInstance method call").isTrue();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceEventsTest.java]: Ditto

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceNameListenerTest.java [hunk 8]
```diff
@@ -49,10 +48,10 @@ public void testProcessCreateProcessNameEvent() throws Exception {
                 name("oneTaskProcessInstanceName").
                 start();
 
-        assertNotNull(processInstance);
+        assertThat(processInstance).isNotNull();
 
-        assertThat("Process instance name must be initialized before PROCESS_CREATED event is fired.",
-                listener.getProcessName(), is("oneTaskProcessInstanceName"));
+        assertThat(listener.getProcessName()).isEqualTo("oneTaskProcessInstanceName")
+                .as("Process instance name must be initialized before PROCESS_CREATED event is fired.");
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceNameListenerTest.java]: Fixed; thanks

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceNameListenerTest.java [hunk 9]
```diff
@@ -68,7 +67,8 @@ public void testCallActivityProcessCreatedDefinitionName() throws Exception {
                 processDefinitionKey("masterProcess").
                 start();
 
-        assertThat("SubProcessInstance PROCESS_CREATED event must have processDefinitionName set", listener.getProcessDefinitionName(), is("Child Process"));
+        assertThat(listener.getProcessDefinitionName()).isEqualTo("Child Process")
+                .as("SubProcessInstance PROCESS_CREATED event must have processDefinitionName set");
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/ProcessInstanceNameListenerTest.java]: Ditto

**Comment by filiphr** [comment]
Thanks for the changes @dbmalkovsky

**Pull request merged**